lte/cloud/go/services/lte/obsidian/models/conversion.go

```diff
@@ -626,3 +627,191 @@ func (m *ApnResource) getAssocs() []storage.TypeAndKey {
 	apnAssoc := []storage.TypeAndKey{{Type: lte.APNEntityType, Key: string(m.ApnName)}}
 	return apnAssoc
 }
+
+func (m *GatewayPool) FromBackendModels(ent configurator.NetworkEntity) error {
+	m.GatewayPoolName = ent.Name
+	m.GatewayPoolID = GatewayPoolID(ent.Key)
+	cfg, ok := ent.Config.(*GatewayPoolConfigs)
+	if !ok {
+		return fmt.Errorf("could not convert entity config type %T to GateawyPool", ent.Config)
+	}
+	m.Config = cfg
+	m.GatewayIds = []models.GatewayID{}
+	for _, gwID := range ent.Associations {
+		m.GatewayIds = append(m.GatewayIds, commonModels.GatewayID(gwID.Key))
+	}
+	return nil
+}
+
+func (m *GatewayPool) ToEntity() configurator.NetworkEntity {
+	assocs := []storage.TypeAndKey{}
```

Comment:
Not relevant here but: `storage.TKs` for some extra slice-based goodies like `.Filter()`